### PR TITLE
fix(ui): pipelineDetails page update

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

I worked on the below changes on PipelineDetails page
- Remove Details tab
- Refactoring pipelineDetails code
- Use localisation where as possible
- Use AntD components and eliminate custom components

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
Screenshot 2022-11-09 at 2 41 58 PM
Screenshot 2022-11-09 at 2 42 13 PM
Screenshot 2022-11-09 at 5 20 20 PM
Screenshot 2022-11-09 at 5 38 09 PM
Screenshot 2022-11-09 at 2 42 47 PM
Screenshot 2022-11-09 at 2 42 55 PM
Screenshot 2022-11-09 at 10 38 39 AM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Execution/TreeView/TreeViewTab.component.tsx [hunk 1]
```diff
@@ -80,25 +79,33 @@ const TreeViewTab = ({
...
                       alt="result"
-                      className="tw-w-6"
+                      className="tw-w-6 mr-2 mb-2"
                       icon={getStatusBadgeIcon(status.executionStatus)}
                     />
                   </Tooltip>
                 ))}
-              </Space>
+              </div>
+              <Space />
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Execution/TreeView/TreeViewTab.component.tsx:108]: do we need this?

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 2]
```diff
@@ -396,16 +452,315 @@ const PipelineDetails = ({
...
+          <Link target="_blank" to={{ pathname: record.taskUrl }}>
+            <Space>
+              <span>{name}</span>
+              <SVGIcons
+                alt="external-link"
+                className="align-middle"
+                icon="external-link"
+                width="16px"
+              />
+            </Space>
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:571]: we can avoid extra wrapping of `<Space>`

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 3]
```diff
@@ -457,139 +812,174 @@ const PipelineDetails = ({
               ? onTierUpdate
               : undefined
           }
-          version={version}
+          version={version + ''}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:815]: is it intentional? if the purpose is to convert to string can we try `${version}`

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx [hunk 4]
```diff
@@ -457,139 +812,174 @@ const PipelineDetails = ({
...
+              </>
+            }>
+            <Card className="h-min-full">
+              <Row justify="center">
+                <Col span={18}>
+                  <div id="activityfeed">
+                    <ActivityFeedList
+                      isEntityFeed
+                      withSidePanel
+                      className=""
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:903]: can we remove the className?

**Review by ShaileshParmar11 — DISMISSED** [review]
Let few comment

File: openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx (hunk 3 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx:815]: yes intentional and required

File: openmetadata-ui/src/main/resources/ui/src/components/Execution/TreeView/TreeViewTab.component.tsx [hunk 5]
```diff
@@ -80,25 +79,32 @@ const TreeViewTab = ({
...
             <Col span={19}>
-              <Space>
+              <div
+                className="execution-node-container"
+                style={{
+                  overflowX: 'auto',
+                  width: '100%',
+                  height: '100%',
+                  whiteSpace: 'nowrap',
+                }}>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Execution/TreeView/TreeViewTab.component.tsx:89]: Remove this.

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by ShaileshParmar11 — DISMISSED** [review]

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**